### ISSUE TITLE
Video se zastaví po krátké chvíli přehrávání

### ISSUE THREAD

**Issue by tslnox** [kodi-czsk/plugin.video.sosac.ph#143]
Zkoušel jsem seriál Krok za krokem - libovolný díl se zastaví po přesně minutě, a film (Ne)lidský kšeft - zastavil se po cca 1,5 minutě. V logu je několik errorů, nejzajímavější pravděpodobně 

ERROR: CCurlFile::FillBuffer - Failed: Requested range was not delivered by the server(33)

Při zkopírování linku a smazání celého textu za .flv (cookie) a otevření ve VLC se stalo přesně totéž (po minutě zastaveno). Při otevření celého linku v prohlížeči skáče 403 Forbidden, bez cookie se načte stránka a video jde stáhnout, pak jde přehrávat ve VLC normálně.

Log zde: https://pastebin.com/S2zbceeL
Zajímavé je, že tentokrát se ten výše uvedený ERROR neobjevil, video ale stejně přestalo hrát přesně po minutě.

**Comment by Japkozloun** [comment]
Přesně stejný problém nastal i u mě. Zkoušel jsem asi 15 různých filmů i seriálů a u všech nastane stejný problém jako ten který byl popsán výše.

**Comment by lukasenkocz** [comment]
https://plus.google.com/u/0/110541347718359311875/posts/XY3vmnCkzb6?cfem=1

**Comment by lzoubek** [comment]
Fixnu plugin tak, aby zobrazil notifikaci v pripade, ze nemas nastaveny premium ucet

**Comment by Saljack** [comment]
Trochu me mrzi ta prace s prepisem na JSON co jsem mel :-( ale dalo se to ocekavat. Myslim, ze byla chyba pridavat moznost premioveho uctu.

**Comment by lzoubek** [comment]
No, ono by bez te moznosti jeli vsechny videa jen tech par minut - holt to borci na streamujtv zmenili. @Saljack tvuj google+ login?

**Comment by Saljack** [comment]
@lzoubek  saljacky at gmail dot com

**Comment by DrazekV** [comment]
A zeptám se bude opět možnost video i stáhnout? Nyní to nejde ani s Premium účtem.

**Comment by sleep-walker** [comment]
zda se, ze uz nejde ani jedna minuta z videa

**Comment by forsajd** [comment]
zdravim poprosim tvocu pluginu ci by nespravil v enigme2 aby sa dalo do sosaca prihlasit cez prihlasovacie udaje ako je to v kodi Dakujem

**Comment by tslnox** [comment]
Stažení celého souboru přes prohlížeč už taky nejde.

**Comment by lzoubek** [comment]
@forsajd to musis kontaktovat cloveka, ktery pluginy na enigmu portuje

**Comment by forsajd** [comment]
dik

**Comment by svacna** [comment]
Bude možné v budoucnu zadat účet k streamuj.tv i v rozšířeném doplňku pro kodi?

**Comment by petrm74** [comment]
Pokud myslíš fork Sosáče od zbyna, tedy http://www.xbmc-kodi.cz/prispevek-fork-sosace-s-rozsirenou-funkcnosti,
tak to se musíš ptát jinde - https://github.com/zbyna/plugin.video.sosac.ph

**Comment by tomaswitek** [comment]
Takze jestli to chapu spravne, musi si uzivatel zaplatit Premium ucet u [Streamuj.cz](http://www.streamuj.tv/_payments.php) a nastavit si pristupove udaje pro streamuj u sosac pluginu?

**Comment by DrazekV** [comment]
Ano přesně tak. Zaplatíš a půjde to. Jen nefunguje stahování jako dřív. Ale přehrávání půjde normálně.

**Comment by tomaswitek** [comment]
OK diky za info @DrazekV

**Comment by Ryesen** [comment]
@DrazekV možná trochu naivně, ale přece: Pokud si přeplatím pouze jeden den, tak mi sosáč bude fungovat pouze předplacenou dobu?

**Comment by DrazekV** [comment]
@Ryesen ano bude fungovat je po dobu platnosti Premium účtu na streamuj.tv po té nebude fungovat.

**Comment by svacna** [comment]
Ano. A také jen na jedné veřejné IP adrese současně.

Dne út 25. 7. 2017 7:35 uživatel DrazekV <notifications@github.com> napsal:

**Comment by svacna** [comment]
Ano také mají kontrolu IP adresy. Na jedné  veřejné IP adrese na x zařízení současně, ale pokud si současně pustí manželka doma film na sosači a ty v práci (na stejném účtu) půjde to jen tomu, kdo to pustí dřív.

**Comment by Ryesen** [comment]
@svacna tak to je úžasné.

Existuje alternativa (česká) pro Sosáč?

**Comment by svacna** [comment]
Zdarma nic.

**Comment by tomaswitek** [comment]
Exodus + Titluky.com je dobra varianta.
Pokud teda vylozene netouzis po dabingu.

**Comment by OrbisPictus** [comment]
Nechcel som sa do toho zapájať a už vôbec propagovať iný plugin (server), ale riešil som toto isté tak viem aké to je. Existuje jedna alternatíva, ktoré je napojená na český fileserver, ktorý ponúka okrem sledovania aj sťahovanie a vyzerá to tak, že nie je viazaný na IP (testované len raz) a za lepšiu cenu. Každopádne ma to celé mrzí, nakoľko som sosáč používal veľmi dlho, ale ponuka streamuj nie je taká atraktívna aby som za ňu platil :( Ak toto prekročí nejaké hranice propagácie :) tak príspevok kľudne vymažte. Viem ako sa Ryesen cíti :)

**Comment by svacna** [comment]
Ano je to škoda, ale na druhou stranu co je dnes 749 za rok? Já myslím, že nic. U jiného doplňku je to asi 550 za rok. Za komfort sledování je to malá částka.

**Comment by Saljack** [comment]
Zdravim panove, tady neni misto, kde by se melo diskutovat o tom kolik co stoji a jaka nahrada existuje. Tohle je issue tracking pro nahlaseni chyby nebo vylepseni. Od diskuze je forum, google+ nebo facebook, kde to muzete probrat. Diky

**Comment by ezdiy** [comment]
Mam rozchozene reseni - je to stejna story jako s megavideem. Takze pekne nastavime v kodi proxy ktera prepisuje URL na zaplaceny login - a jede to vsem na mej lokalni siti.

@lzoubek Proxy pro celou kodi je trochu tezkopadne (nezajima me traffic z ostatnich rozsireni). Bylo by mozne pridat option pro zadani STREAMUJ_URL, at si tam lidi muzou zadat proxy *jen* pro streamuj?

Ad jak nastavit nginx proxy je cviceni pro ctenare a sirit se otom nebudu, ale vypada to ze nekdo holt platit ucet musi, nicmene ho lze do jiste miry sdilet. Tady se slozilo 40 lidi, tzn 20kc na rok - coz neni zas takova hruza.

**Comment by iChUdY** [comment]
@Saljack nesouhlasím. Tato událost je jednoznačně issue !

**Comment by sleep-walker** [comment]
@ezdiy zjevne spravne reseni, jen vyzadujici dostupnou proxynu, pres kterou muze protekat tolik dat... diky za potvrzeni.

**Comment by ezdiy** [comment]
@iChUdY btw, ty si muzes nastavit proxy na 10.77.77.88:8080 :D

**Comment by ezdiy** [comment]
Kdo umite, tak si zkuste:

https://github.com/kodi-czsk/script.module.stream.resolver/pull/60
https://github.com/kodi-czsk/plugin.video.sosac.ph/pull/144

**Comment by iChUdY** [comment]
@lzoubek @Saljack čili přidání premium do pluginu a následné vypnutí bez premium je čistá náhoda ? Sorry jako xD

**Comment by lzoubek** [comment]
@iChUdY ne, to neni nahoda. Lidi ze sosace me kontaktovali v dobe, kdy davali na web testovaci HTML 5 videa a poskytnuli API k tomu, aby bylo mozne poslat do url videa streamujtv jmeno/heslo. Samozrejme jsem je mohl poslat nekam, ale pak by sosac na kodi pravdepodobne uplne skoncil, takze lepsi neco nez nic

@ezdiy tvoje PR mergnu

**Comment by DrazekV** [comment]
@izoubek a zeptám se stahování ještě v budoucnu půjde? :)

**Comment by Saljack** [comment]
@DrazekV urcite by melo jit pokud je rozbite. Ted mam malo casu, ale az budu mit chvilku, tak na to mrknu. Zaloz issue

**Comment by Ryesen** [comment]
Existuje už někde sepsaný návod/postup jak sosáč rozchodit? Dle pročtení tématu nejspíše skrze VPN.

**Comment by DrazekV** [comment]
@Ryesen stačí zaplatit a jedeš jako dřív. Máš to nejjednodušší.

**Comment by Podly27** [comment]
Ahoj Po zaplaceni u vsech filmu tyto hlasky : Nelze ziskat videolink.... a Jednu nebo vice polozek se nepodarilo najit ...

**Comment by sleep-walker** [comment]
@Podly27 to asi nesouvisi s puvodnim problemem, zaloz si nove issue (dnes jsou zdarma).
